hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseOptions.java

```diff
@@ -137,4 +137,19 @@ public static synchronized HbaseOptions instance() {
                     null,
                     ""
             );
+    public static final ConfigOption<String> HBASE_VERTEX_TABLE_REGION_NUM =
+            new ConfigOption<>(
+                    "hbase.hbase_vertex_table_region_num",
+                    "The HBase's pre-splite region num.",
```

Comment:
can improve the description like:
"The number of partitions of the HBase vertex table, it can be set to the number of pre-split regions."